Tribler/Core/Modules/MetadataStore/store.py

```diff
@@ -34,14 +33,13 @@
 # maintained by SQL triggers.
 sql_create_fts_table = """
     CREATE VIRTUAL TABLE IF NOT EXISTS FtsIndex USING FTS5
-        (title, tags, content='ChannelNode', prefix = '2 3 4 5',
+        (title, content='ChannelNode', prefix = '2 3 4 5',
```

Comment:
Do we have to remove our metadata database to run this?

Reply:
Actually, we don't have to. But it is a very good idea to do so because your FTS index gets much more balanced afterward.

Reply:
Ok, now I've broken the schema. So, we have to remove our database.